### DISCUSSION TITLE
Comments About the Survey for the DEIC-CoDADE Position Paper

### DISCUSSION THREAD

**Discussion by aramdraper** [aiaa-deic/computation-ai#22] [General]
Please add all comments from reviewing the [survey](https://github.com/aiaa-deic/computation-ai/blob/main/DEIC-CAE%20Survey%20Questions%20(Updated%2008-07-2024).docx) under this thread. Thank you!

**Comment by arthurrge** [comment]
I'd cycled the questions with Joe Walsh (NAFEMS ASSESS) - summarizing his comments here (incorporating my own takes)

General comment from Joe: Too long!

Q5: Add one level between team & enterprise (Perhaps call it “division-wide” and add (engineering, manufacturing, etc.)
Q7: Drop. Unclear this adds much value vs. awkwardness.
Q8: Add definitions (I see in the latest version the up-front does some of this) - I also thought perhaps what we mean to ask here is not "agree with" but "is how you apply twin/thread aligned with the definitions" - would be nice then to have some idea of the misalignment, thought that may be for those teams to work out.
Q13: Split enterprise vs. collaborators (distinct)
Q21-27: Add metadata as an option 
Q30: Responders will probably not know the answer - some who do may not want to answer.  Perhaps something better here around dedicated vs. on-demand and sufficiency of system / budget for desired analyses.
Q37-39: Add regulatory bodies.
Q40: add "no recent cost-cutting" (which is different than no impact)

**Comment by arthurrge** [comment]
Comments for discussion:

1.	I still think we have to carefully walk the knife edge between AI and not-AI. AI is good for getting attention and being “at the table” for those important discussions in the context of Digital Engineering. However, this is a bit of a bait and switch, as AI is but one of a multitude of computational methods, and AI is not even “new” as such. (Though this comes down to semantics in how we scope “AI”). The AI aspect does speak more to the “responsible use” part of the title, but I’m concerned we may be neglecting many, many, many crucial aspects of “computational methods” { multiscale, coupled physics, scalability, realism (non-simplification), fidelity/accuracy/VVUQ, software maturity, etc. } in addition to the very key interoperability (/flexibility/modularity/extensibility/etc) implied in the title/description.
 
2.	Non-deterministic Methods: The concern I keep expressing over this term is that it has a pretty specific meaning in embedded systems, relating to reliably repeatable (verbatim verifiable) system behavior – and the sense in which it is used in the description/discourse is much more broad than that – relating more to data-derived (learned) or probabilistic (stochastic) models alone or in conjunction with (presumably “deterministic”) rules-encoded (physics/etc) models. There are many important interoperability aspects for the discussion in that sense – combining measured + synthetic data, the multiscale integration problems, high-order solver effects, parallel-in-time effects, and even mundane things like data formatting, metadata taxonomies, etc. So while the “outside of the box” can read as below just fine, I would continue to encourage the “inside the box” content to be fortified with such vitamins and minerals. 😊

**Comment by arthurrge** [comment]
Survey completed 4Q2024.
